src/libpriv/rpmostree-importer.c

```diff
@@ -548,6 +548,18 @@ path_for_tmpfiles_should_be_ignored (const char *path)
   return g_str_has_prefix (path, "/var/lib/rpm");
 }
 
+/* systemd-tmpfiles complains loudly about writing to /var/run; ideally,
+ * all of the packages get fixed for this but...eh.
+ */
+static void
+append_translated_tmpfiles_path (GString *buf, const char *path)
+{
+  static const char varrun[] = "/var/run/";
+  if (g_str_has_prefix (path, varrun))
+    path += strlen (varrun) - 1;
```

Comment:
Shouldn't this be something like `path += strlen("/var");`? Otherwise we also trim the `/run` bit, no?

Reply:
Hah, yes thanks!  Will fix, looking at tests.